File: ReactWebApp/src/components/TankkausData.js
import React from 'react'
import '../App.css'
import { Button } from 'react-bootstrap'

const TankkausData = ({ tankkaukset }) => tankkaukset.map(t =>


    <tr key={t.tankkausId}>
        <td>{t.pvm}</td>
        <td >{t.reknro}</td>
        <td>{t.mittarilukema}</td>
        <td>{t.euroa}</td>
        <td>{t.litraa}</td>
        <td>{t.ajomaara}</td>
        <td>{t.keskikulutus}</td>
        <Button>poista</Button>
    </tr>
)

export default TankkausData

File: Core-RestAPI/Models/TankkausData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TimesheetRestApi.Models
{
    public class TankkausData
    {
        public int TankkausId { get; set; }
        public DateTime? Pvm { get; set; }
        public decimal? Litraa { get; set; }
        public decimal? Euroa { get; set; }
        public string Reknro { get; set; }
        public int Mittarilukema { get; set; }
        
    }
}

